gateways/core/stream/amqp/config.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package amqp
 
 import (
+	"errors"
```

Comment:
Would you mind changing it to `github.com/pkg/errors` to keep the codebase consistent